docs/source/en/tasks_explained.mdx

```diff
@@ -0,0 +1,230 @@
+
+The main change ViT introduced was in how images are processed:
+
+1. An image is split into square patches. If you had a 224x224 pixel image, you could split it into 256 14x14 image patches. Just like how text is tokenized into words, an image is "tokenized" into a sequence of patches. Each patch is flattened by concatenating the height and width pixel values into a single vector. For a 3D image, the height, width, and number of *channels* are flattened.
+
+    The pixel values are linearly transformed into *patch embeddings* by a 2D convolution to create the proper input dimensions. Image patches that are similar are mapped to similar patch embeddings.
+
+2. A *learnable embedding* - a special `[CLS]` token - is added to the beginning of the patch embeddings just like in the BERT model. The encoder output with the `[CLS]` token is used as the input to the attached classification head; other outputs are ignored. This token helps the model learn how to encode a representation of the image.
+
+3. The last thing to add to the patch and learnable embedding is the *position embedding* because the model doesn't know how the image patches are ordered. The position embeddings are also learnable and have the same size as the patch embeddings. Finally, all of the embeddings are passed to the Transformer encoder.
+
+4. The output, specifically only the output with the `[CLS]` token, is passed to a multilayer perceptron head (MLP). ViT's pretraining objective is simply classification. Like other classification heads, the MLP head converts the output into logits and calculates the cross-entropy loss to find the most likely label.
+
+Ready to try your hand at image classification? Check out our complete [image classification guide](tasks/image_classification) to learn how to finetune ViT and use it for inference!
+
+#### ConvNet
+
+[ConvNeXT](model_doc/convnext) is a CNN architecture that adopts new and modern network designs to improve performance. However, convolutions are still at the core of the model. From a high-level perspective, a [convolution](glossary#convolution) looks at a small window of the image pixels and computes some features from it such as a certain texture or curvature of a line. The distance the convolution travels is known as the *stride*. You can feed this output to another convolutional layer, and with each successive layer, the network learns more complex and abstract things like hotdogs or rockets. Between convolutional layers, it is common to add a pooling layer to reduce dimensionality and make the model more robust to variations of a feature's position. This is the basic idea behind CNNs.
+
+From here, ConvNeXT modernizes the CNN in five ways:
+
+1. Change the number of blocks in each stage and "patchify" an image with a larger stride and corresponding kernel size. The non-overlapping sliding window makes this patchifying strategy similar to how ViT splits an image into patches.
```

Comment:
You explain what stride is but this is the first mention of kernel. I'm not sure if it's better to add a definition of kernel, or to remove mention of both for simplicity.

Reply:
Added a brief explanation of kernel so we can use this vocabulary and especially since it comes up a lot in computer vision